scripts/Muon/GUI/ElementalAnalysis/elemental_analysis.py

```diff
@@ -90,6 +90,8 @@ def __init__(self, parent=None):
 
         # plotting
         self.plot_window = None
+        self.colors = ['b', 'g', 'r', 'y', 'c', 'm', 'b']
```

Comment:
Is it worth using the default color cycle for the lines, `mpl.rcParams['axes.prop_cycle']`, so that it will follow the default matplotlib line colors and match the standard plots? https://matplotlib.org/2.2.4/users/dflt_style_changes.html#colors-color-cycles-and-color-maps

Reply:
As this was merged then I guess this can be a new issue.